src/main/java/com/endava/endabank/dto/user/AuthenticationDto.java

```diff
@@ -1,10 +1,26 @@
 package com.endava.endabank.dto.user;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
 
 @Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
 public class AuthenticationDto {
+    @Email
+    @NotNull
```

Comment:
Hi,
Luis the validations has a default message in this case it will return, Email format no valid, and for the null anotation it will return, "nameField" cannot be null